### ISSUE TITLE
How to extract features?

### ISSUE THREAD

**Issue by kidpaul94** [yanx27/Pointnet_Pointnet2_pytorch#107]
Thank you for your work @yanx27!
Could you tell me where I can get feature points (from Hierarchical point set feature learning) in this pytorch version?

**Comment by yanx27** [comment]
@kidpaul94 Hi, thanks for your attention. You can return `l1_points`, `l2_points` and `l3_points` from PointNet++ [here](https://github.com/yanx27/Pointnet_Pointnet2_pytorch/blob/ba4c5cf3a1e2b735d696c59a43c38345c3d003b9/models/pointnet2_cls_ssg.py#L29).

**Comment by kidpaul94** [comment]
Thank you for the reply! I'll try it.

**Comment by karimaly29** [comment]
Hii thank u so much for ur work @yanx27 .. i'm running the test_classification and i wanted to extract the features .. i added the return l1_points, l2_points and l3_points from PointNet++ to the pointnet2_cls_ssg.py but when i run the test_classification using a pretrained model (checkpoints), i think it doesn't run the pointnet2_cls_ssg.py again, cause i don't get any features, however i got the code running and an accuracy etc.  .. how could i extract the features? thank u in advance!
